grails-app/views/inventoryItem/_showCurrentStock.gsp

```diff
@@ -62,17 +64,26 @@
                     <td>
                         <g:expirationDate date="${entry?.inventoryItem?.expirationDate}"/>
                     </td>
-
                     <td>
-                        ${entry?.inventoryItem?.comments}
+                        ${g.formatNumber(number: entry?.quantityOnHand, format: '###,###,###') }
+                        ${entry?.inventoryItem?.product?.unitOfMeasure}
                     </td>
                     <td>
-                        ${g.formatNumber(number: entry?.quantity, format: '###,###,###') }
-                        ${entry?.product?.unitOfMeasure}
+                        ${g.formatNumber(number: entry?.quantityAvailable, format: '###,###,###') }
+                        ${entry?.inventoryItem?.product?.unitOfMeasure}
+                    </td>
+                    <td>
+                        <g:if test="${entry?.status == AvailableItemStatus.PICKED}">
+                            <%-- TODO: Change the selected tab to pending outbound --%>
```

Comment:
@jmiranda do you know how I could change the tab from here?

Reply:
Could you try adding different id to each tab and add an onclick event here which will find element (tab) by id and perform a `.click()` on it?

Reply:
It's working, thanks @awalkowiak